### DISCUSSION TITLE
production use

### DISCUSSION THREAD

**Discussion by gmesika-coti** [cosmos/community#16] [Chain Builders (Cosmos SDK)] [answered]
what should be used for running production node?

https://docs.cosmos.network/v0.46/run-node/run-node.html#
or 
https://tutorials.cosmos.network/hands-on-exercise/1-ignite-cli/1-ignitecli.html

**Comment by gmesika-coti** [comment]
meanwhile, I'm trying to get more infromation about this.
some of the big projects running on cosmos (kava, osmosis) don't seems to use the ignite but should be built using the make build.
so I should be replacing the binary name, like in cosmos-sdk instructions:

Make sure you can build your own binary, and replace simd with the name of your binary in the snippets.

how do I change the binary name so it wont product the simd but some other name like projectd?, and is that the right way to go?

**Reply by julienrbrt** [reply]
Yes, people are using Ignite for the boilerplate, but often chain move away from it to gain flexibility.
The cobra root command determines the application name, and go build just use the directory name of the main package for the default binary name (the actual file you are getting).

**Comment by julienrbrt** [comment, marked as answer]
Hi, the first link you've posted is a good resource to help you to run a node.
Additionally, this link as well as good practices: https://tutorials.cosmos.network/tutorials/9-path-to-prod/1-overview.html

**Comment by Fardonal** [comment]
Hi.Thanks for the detailed instructions,it has now helped me a lot to understand a few details of Cosmos-SDK and getting the node up and running.
